code/modules/tutorials/blob_tutorial.dm

```diff
@@ -735,7 +736,7 @@ proc/AddBlobSteps(var/datum/tutorial/blob/T)
 	T.AddStep(new /datum/tutorialStep/blob/cutscene3)
 	T.AddStep(new /datum/tutorialStep/blob/finished)
 
-/obj/blob_tutorial_walker
+/mob/blob_tutorial_walker
```

Comment:
why?

Reply:
Didn't see why flamethrowers should handle being invoked by arbitrary objects. 

Alternatives:
A) Adding a property to that object.
B) Gracefully handle lack of object property at caller.

Reply:
It's just weird creating an entirely new mob basetype just for this, furthermore it's not like it uses any mob functionality.

Reply:
shouldn't be hard to add some checks in flamer code to handle being fired without `mob/user`

Reply:
Thanks! Ended up turning into a great lesson in type safety...